src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -300,15 +292,15 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
           tabindex=${ifDefined(
             !hasAction(entityConf.tap_action) ? "-1" : undefined
           )}
-          .disabled=${!hasAction(entityConf.tap_action)}
           .config=${entityConf}
           class=${classMap({
-            "state-on": !STATES_OFF.has(stateObj.state),
+             "state-on": stateActive(stateObj),
           })}
           title=${`${computeStateName(
             stateObj
           )} : ${this.hass.formatEntityState(stateObj)}`}
         >
+        .disabled=${!hasAction(entityConf.tap_action)}
```

Comment:
This is not even part of the original element and an obvious error. Just put it back in the same line you removed it from.

It's great that you would like to fix bugs and increase your contributions, but at the same time you should really take a pause and take the time to setup an appropriate development environment where you can run the linters, and more importantly test your changes.

At the moment it just seems like you're slapping things around at random without testing them, and I think that will quickly wear out the patience of your reviewers :)